frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.ts

```diff
@@ -137,35 +132,37 @@ export class NewMeetingComponent extends BaseComponent implements OnInit {
         });
     }
 
-    showUnshowCustomDuration(form: FormGroup) {
-        const durationValue = form.value.duration;
+    onDurationChange(form: FormGroup) {
+        this.duration = form.value.duration;
 
-        this.customTimeShown = durationValue.time === 'Custom';
-        this.mainContentCustomTimeShown = form.value.mainContainerDuration.time === 'Custom';
+        this.customTimeShown = this.duration.time === 'Custom';
 
         if (this.customTimeShown) {
             this.setValidation();
         } else {
-            this.durationChanged(durationValue.time, durationValue.unitOfTime);
+            this.meetingForm.controls['customTime'].setValue('');
         }
     }
 
     customDurationChanged(form: FormGroup) {
-        const { customTime, unitOfTime } = form.value;
+        const { customTime } = form.value;
 
-        this.durationChanged(customTime, unitOfTime);
-    }
+        const unitOfTime: UnitOfTime = form.controls['unitOfTime'].value;
+        let customMinutes: number;
 
-    durationChanged(timeValue: string, unitOfTime: string) {
-        if (unitOfTime === 'hour') {
-            this.convertDuration(timeValue);
+        if (unitOfTime === UnitOfTime.Hour) {
+            customMinutes = parseInt(customTime, 10) * 60;
         } else {
-            this.duration = parseInt(timeValue, 10);
+            customMinutes = parseInt(customTime, 10);
```

Comment:
![image](https://user-images.githubusercontent.com/38536017/188319422-7769c990-0acb-4a6a-ae33-48bb473c8591.png)
you can combine them
Use ternary operator